api/modules/files/routes/read.js

```diff
@@ -1,25 +1,27 @@
-var Joi = require('joi');
+"use strict";
 
-var Errors = require('../../../classes/errors');
-var prereqs = require('../../../classes/prerequisites');
+const Joi = require(`joi`);
 
-var controller = require('../controller');
-var Model = require('../model');
+const Errors = require(`../../../classes/errors`);
+const Prerequisites = require(`../../../classes/prerequisites`);
+
+const FilesModel = require(`../model`);
+const filesController = require(`../controller`);
 
 module.exports = [{
-  method: 'GET',
-  path: '/files/{id}',
+  method: `GET`,
+  path: `/files/{id}`,
   config: {
     pre: [
-      prereqs.confirmRecordExists(Model, {
-        mode: 'param',
-        requestKey: 'id'
+      Prerequisites.confirmRecordExists(FilesModel, {
+        mode: `param`,
+        requestKey: `id`
       }),
-      prereqs.validateUser(),
-      prereqs.validateOwnership()
+      Prerequisites.validateUser(),
+      Prerequisites.validateOwnership()
     ],
-    handler: controller.getOne.bind(controller),
-    description: 'Retrieve a single file object based on `id`.',
+    handler: filesController.getOne.bind(filesController),
```

Comment:
`handler: args => filesController.getOne(...args),` although it feels odd that this can't just be a static function and needs explicit tying to a specific fileController instance. It suggests we might have unnecessary data capturing in the fileController class.